lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -38,5 +41,30 @@ public PuppeteerLaunchTests()
 
             await browser.CloseAsync();
         }
+
+        [Fact]
+        public async Task NetworkRedirectsShouldReportSecurityDetails()
+        {
+            var options = TestConstants.DefaultBrowserOptions();
+            options.IgnoreHTTPSErrors = true;
+
+            var browser = await PuppeteerSharp.Puppeteer.LaunchAsync(options, TestConstants.ChromiumRevision);
+            var page = await browser.NewPageAsync();
+
+            // TODO:
+            // httpsServer.setRedirect('/plzredirect', '/empty.html');
```

Comment:
not sure how to do this

Reply:
The test project is using the TestServer to test a local website. You should run `dotnet ./bin/Debug/netcoreapp2.0/PuppeteerSharp.TestServer.dll` to get the server running.

As the TestServer is an ASP.NET Core project, you could create there a Route `plzredirect` (or a better name) which redirects to `empty.html`.